src/utils/SparseMatrixMultiplyMod.F90

```diff
@@ -1243,7 +1243,7 @@ subroutine SPMP_ABC(this,num_unit,filter_u,A,B,C,list_ready,list_A,list_B,list_C
           call endrun( subname//" ERROR: missing required optional arguments" )
           return
        end if
-       SHR_ASSERT_FL((size(filter_actunit_C) > num_actunit_C), sourcefile, __LINE__)
+       SHR_ASSERT_FL((size(filter_actunit_C) >= num_actunit_C), sourcefile, __LINE__)
```

Comment:
From meeting with @ekluzek 
Search for other ASSERT lines with "size(...) > num" and replace with >=